exchanges/exchange.go

```diff
@@ -47,6 +47,8 @@ var (
 
 	errEndpointStringNotFound = errors.New("endpoint string not found")
 	errTransportNotSet        = errors.New("transport not set, cannot set timeout")
+
+	ErrPairNotFound = errors.New("pair not found")
```

Comment:
Comment please or un-export.

Reply:
this is actually being used [here](https://github.com/thrasher-corp/gocryptotrader/pull/869/files#diff-e17060c728c86d82e2bb7909ca3677a3af1c272d317f799dbe9a3c37dc13c2f1R232) to actively ignore this specific error and none other